1-exercises/B-while-loop/exercise.js

```diff
@@ -6,7 +6,14 @@
 */
 
 function evenNumbers(n) {
-    // TODO
+  // TODO
+  let array = [];
+  let even = 0;
```

Comment:
Hi Lorena, excellent coding! I am super impressed. 
Just a very minor suggestion that you can create a variable that starts as being equal to 0 and reflects the amount of times you have executed the loop, just as an extra check. I did it this way but let me know what you think as it is not necessarily needed.